.github/workflows/holonix-cache.yml

```diff
@@ -20,6 +20,7 @@ jobs:
           - ''
           - '--refresh --override-input versions github:holochain/holochain?dir=versions/0_1'
           - '--refresh --override-input versions github:holochain/holochain?dir=versions/0_2'
+          - '--refresh --override-input versions github:holochain/holochain?dir=versions/rc'
```

Comment:
Hmm, that was my intention to just have it pointing to the release candidates for the next release version of Holochain. The minor version changing I think is kind of acceptable because between those release candidates there should have been a stable release for people to switch back to. If we wanted to be able to put out release candidates on multiple minor versions at once we'd be a bit stuck though..

Yeah this makes sense, I'll go ahead with the naming change